### DISCUSSION TITLE
Как правильно обновится на 2021.3.86

### DISCUSSION THREAD

**Discussion by enigmen** [mikopbx/Core#195] [Q&A] [answered]
Доброго времени, подскажите пожалуйста каков порядок обновления на 2021.3.86 с версией 2019.4.53? Используется панель телефонии 1.4.26.30 и модуль журнала звонков и аналитика.

**Comment by boffart** [comment, marked as answer]
Нужно обновлять и АТС и панель телефонии для 1С. 
Все до актуальной версии. 
Бекап обязателен. 

1. В идеале, если используется виртуальная машина, сделать снэпшот. 
2. После этого обновить АТС. 
3. Проверить работают ли входящие. / исходящие
4. Проверить работает ли интеграция с 1С. 
5. Если все ок, то можно на этом остановиться
6. Если не работает интеграция с 1С, то бекапите базу 1С, и обновляете панель телефонии
7. Если проблемы остаются, то откатываетесь до бекап версии и пишите на форум описание ситуации
